muhasin/index.js

```diff
@@ -0,0 +1,15 @@
+let str = "it's hard to say who is right in this matter"
+// Find the first occurrence of the letter "a" in a string, starting the search at position 17.
+console.log("str ==>", str.indexOf("a")); // 6
+console.log("str ==>", str[17]); // w
```

Comment:
Wrong answer! Try this one instead to find the index of a (starting at position 17)
console.log("str ==>", str.indexOf("a",17)); // 39